src/_test_/task.test.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import Task from '../classes/task';
+
+describe('Add and remove task', () => {
+  test('Add new task', () => {
+    const res = new Task();
+    res.addTask("Don't forget Cigars!");
+
+    expect(res.tasks.length).toBe(1);
+  });
+  test('Check equality', () => {
+    const expected = [
+      { index: 0, description: "Don't forget Cigars!", completed: false },
+      { index: 1, description: "Don't forget Milk!", completed: false },
+    ];
+    const res = new Task();
+    res.addTask("Don't forget Milk!");
+    expect(res.tasks).toStrictEqual(expected);
+  });
+  test('Remove Task', () => {
+    const expected = [
+      { index: 0, description: "Don't forget Cigars!", completed: false },
+    ];
+    const res = new Task();
+    res.deleteTask(1);
+    expect(res.tasks).toStrictEqual(expected);
+  });
```

Comment:
* I suggest you use mocks for these functions which updates the local storage and manipulates the DOM [Refer to the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_1.md#:~:text=Some%20of%20those,in%20the%20DOM.) as it is required for this project

Reply:
- [ ] This was not addressed as requested by the previous reviewer please use mock for functions that used localStorage, DOM
and submit another CRR